.github/workflows/test.yml

```
@@ -75,9 +75,9 @@ jobs:
         run: |
           git tag
           pip install --upgrade pip
-          pip install -U setuptools setuptools_scm wheel twine check-manifest
+          pip install -U setuptools twine check-manifest
```

```suggestion
pip install -U build twine check-manifest
```

thanks,  we need `build` though ... not setuptools (build will create an env and grab setuptools based on pyproject.toml)